internal/providers/pluginfw/products/app/resource_app.go

```diff
@@ -128,7 +189,7 @@ func (a *resourceApp) Read(ctx context.Context, req resource.ReadRequest, resp *
 		return
 	}
 
-	var newApp apps_tf.App
+	var newApp appResource
 	resp.Diagnostics.Append(converters.GoSdkToTfSdkStruct(ctx, appGoSdk, &newApp)...)
```

Comment:
When we read it back, won't it be reset to `false`?

Reply:
Excellent eye. This is causing integration tests to fail. I think it is set to `null` but otherwise you're exactly right. I need to set this explicitly in the `Read()` method.